packages/office-ui-fabric-react/src/components/FocusTrapZone/examples/FocusTrapZone.Nested.Example.tsx

```diff
@@ -86,6 +83,11 @@ export class FocusTrapZoneNestedExample extends React.Component<{}, IFocusTrapZo
     this.setState({ activeStates: { ...activeStates, [zoneNumber]: isActive } });
   };
 
+  // This randomize example is exposing a quirk in focus stack behavior.
+  // Since for the example components render from the bottom up, the most recently active item in the focusStack
+  // ends up being the highest parent, which is the reverse order focus trap zones would normally be put on the
+  // focusStack. That means children aren't capturing focus as one would normally expect when toggling the FTZ's
+  // individually. This would also be an issue if anyone ever rendered multiple FocusTrapZones simultaneously.
```

Comment:
Suggestion:

```suggestion
  // individually. This would also be an issue if anyone ever rendered multiple nested FocusTrapZones simultaneously.
```